README.md

```diff
@@ -1,177 +1,29 @@
-| `not-contains-crds` | Check whether the Helm chart does not include CRDs.
+The tool provides the following features:
 
-The following checks are being implemented and/or considered:
+-   Helm chart Verification: Verifies if a Helm chart is compliant with a certain set of independent checks with no particular execution order.
+-   Red Hat Certified chart validation: Verifies the Helm chart's readiness for being certified and submitted in the Red Hat Helm Repository.    
+-   Report generation: Generates a verification report in the YAML format.    
+-   Customizable checks: Defines the checks you wish to execute during the verification process.
 
-| Name | Description
-|---|---
-| `keywords-are-openshift-categories` | Checks whether the Helm chart's `Chart.yaml` file includes keywords mapped to OpenShift categories.
-| `is-commercial-chart` | Checks whether the Helm chart is a Commercial chart.
-| `is-community-chart` | Checks whether the Helm chart is a Community chart.
-| `not-contains-infra-plugins-and-drivers` | Check whether the Helm chart does not include infra plugins and drivers (network, storage, hardware, etc)
-| `can-be-installed-without-manual-prerequisites` |
-| `can-be-installed-without-cluster-admin-privileges` |
+For further information:
 
-## Architecture
+[The command line interface and checks performed.](docs/helm-chart-checks.md)
 
-This tool inspects a Helm
-chart URI (`file://`, `https?://`, etc)
-and returns either a *positive* result indicating the Helm chart has passed all checks, or a *negative* result indicating
-which checks have failed and remedial actions.
+[Annotations in the report, tool and submitter provided.](docs/helm-chart-checks.md)
 
-The application is separated in two pieces: a command line interface and a library. This is handy because the command
-line interface is specific to the user interface, and the library can be generic enough to be used to, for example,
-inspect Helm chart bytes in flight.
+[Introduction to the submission process.](docs/helm-chart-submission.md)
 
-One positive aspect of the command line interface specificity is that its output can be tailored to the methods of
-consumption the user expects; in other words, the command line interface can be programmed in such way it can be
-represented as either *YAML* or *JSON* formats, in addition to a descriptive representation tailored to human actors.
+[Checks are failing? Try the trouble shooting guide.](docs/helm-chart-troubleshooting.md)
 
-Primitive functions to manipulate the Helm chart should be provided, since most checks involve inspecting the contents
-of the chart itself; for example, whether a `README.md` file exists, or whether `README.md` contains the `values`'
-specification, implicating in offering a cache API layer is required to avoid downloading and unpacking the charts for
-each test.
+For Developers:
 
-## Getting chart-verifier
+[Additional info for developers.](docs/helm-chart-devlopers.md)
```

Comment:
Suggest:Additional info for developers/Contribute towards the development of this tool